sdk/cosmos/azure_data_cosmos/CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Release History
 
-## 0.23.0 (Unreleased)
+## 0.22.1 (Unreleased)
```

Comment:
Probably worth just updating the CHANGELOG.md for the impending release here now.

Reply:
Done